lib/Doctrine/Common/Collections/Collection.php

```diff
@@ -248,6 +261,22 @@ public function forAll(Closure $p) : bool;
      */
     public function map(Closure $func) : self;
 
+    /**
+     * Applies iteratively the given function to each element in the collection,
+     * so as to reduce the collection to a single value.
+     *
+     * @param mixed $initial
+     *
+     * @return mixed
+     *
+     * @psalm-template TReturn
+     * @psalm-template TInitial
+     * @psalm-param Closure():TReturn $func
```

Comment:
Parameters are required to specify. Please make sure whole definition works in psalm.dev https://psalm.dev/r/61a6394fe6